src/inputProcessor.js

```diff
@@ -91,4 +94,12 @@ function setInlineStyles(svg, emptySvgDeclarationComputed) {
   }
 }
 
+function setInternalStyles(svg) {
+  const style = document.createElement('style')
+  style.innerHTML = [...document.styleSheets]
```

Comment:
It's more of personal preference, but I find `Array.from` to be more expressive.